i2c/mcp9808_i2c/README.adoc

```diff
@@ -0,0 +1,37 @@
+= Attaching a MCP9808 digital temperature sensor via I2C
+
+This example code shows how to interface the Raspberry Pi Pico to the MCP9808 digital temperature sensor board. 
+======
+This example reads the ambient temperature value each second from the sensor and sets upper, lower and critical limits for the temperature and checks if alerts need to be raised. The CONFIG register can also be used to check for an alert if the critical temperature is surpassed.
+======
+
+== Wiring information
+
+Wiring up the device requires 7 jumpers, to connect VDD, GND, SDA, SCL, A0, A1 and A2 pins. An additional 2 jumpers could also be required to connect the Pico GND and Pico VSYS pin to the power rail on the breadboard. The example here uses I2C port 0, which is assigned to GPIO 4 (SDA) and 5 (SCL) in software. Power is supplied from the VSYS pin.

```

Comment:
"An additional 2 jumpers could also be required..."
It's probably less confusing to a beginner if you _only_ document the wiring that you've used in the wiring diagram, rather than offering them alternatives?

Reply:
All done! Requires 9 jumpers now